src/backend/api/meals.js

```diff
@@ -1,15 +1,63 @@
-import express from "express";
+import express, { response } from "express";
```

Comment:
I think the `response` import here is accidental as it is unused. 😄

Reply:
thank you, fixed this 👀